automation/vars/uni06zeta.yaml

```diff
@@ -0,0 +1,68 @@
+---
+vas:
+  uni06zeta:
+    stages:
+      - path: examples/dt/uni06zeta/control-plane/nncp
+        wait_conditions:
+          - >-
+            oc -n openstack wait nncp
+            -l osp/nncm-config-type=standard
+            --for jsonpath='{.status.conditions[0].reason}'=SuccessfullyConfigured
+            --timeout=60s
+
+          - >-
+            oc label node master-0 openstack.org/cinder-lvm=''
```

Comment:
this must go in a post_run hook, using @psathyan new capability offered with https://github.com/openstack-k8s-operators/ci-framework/pull/1388

Reply:
Also fails validation, moving to hostname selector.

Reply:
also less call ;-)